sdk/python/tests/vectors/all.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, facade_class, network_class):
 		self.verifier_class = self.facade_class.Verifier
 		self.network_class = network_class
 		self.address_class = self.facade_class.Address
+		self.shared_key_interface = self.facade_class.SharedKey
```

Comment:
why interface instead of class in the name like all the others ?

Reply:
it feels weird to call it class, when it's _not_ a class per se 😬  (I mean you can't use it to make any instances)

Reply:
i see what you mean because it only contains statics.

not sure,  but what if you had some construction like 
`SharedKey(key_pair).derive(public_key)`

Reply:
although, i'd still call it class due to usage of class keyword ^^